flask_resty/jwt.py

```diff
@@ -89,3 +96,53 @@ def get_config_key(self, key):
 
     def get_credentials(self, payload):
         return payload
+
+
+class JwkSetAuthentication(JwtAuthentication):
+    def __init__(self, key_set=None, **kwargs):
+        super(JwkSetAuthentication, self).__init__(**kwargs)
+
+        self.jwks = key_set
+        self.algorithms = get_default_algorithms()
+
+    def get_jwks(self):
+        config = flask.current_app.config
+        return self.jwks \
+            if self.jwks else config[self.get_config_key('key_set')]
```

Comment:
need to add key_set to `JWT_DECODE_ARG_KEYS`?

Reply:
no i think this is ok